app/controllers/jobseekers/profiles/professional_body_memberships_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Jobseekers::Profiles::ProfessionalBodyMembershipsController < Jobseekers::ProfilesController
+  helper_method :jobseeker_profile, :form, :professional_body_membership
```

Comment:
I'm nit a fan of this helper_method pattern. Can we not just instantiate the form in a controller variable? Stops it being used everywhere in the view w/o the @ to show that we're using a  variable. WDYT?